### PR TITLE
chore: fix Operator upgrade tests for OpenShift 4.6 and above

### PR DESCRIPTION
**Pull request by ivanstanev**
- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

Fix Operator upgrade tests for OpenShift 4.6 and above.

### More information

- [Jira ticket RUN-1393](https://snyksec.atlassian.net/browse/RUN-1393)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -462,27 +453,23 @@ jobs:
...
-                    set +x
-                    operator-courier push "${OPERATOR_DIR}" "${QUAY_NAMESPACE}" "${PACKAGE_NAME}" "${LATEST_TAG}" "${QUAY_TOKEN}"
-                    set -x
-                name: Package Operator Bundle upgrade and push to Quay
+                    python scripts/operator/create_operator_bundle_and_index_and_push.py $OPERATOR_DIR $LATEST_TAG $DOCKERHUB_USER $DOCKERHUB_PASSWORD $REPLACES_VERSION
+                description: Now we are testing that any upgrades to the Operator are detected and applied in the cluster.
+                name: Package Operator Bundle upgrade and push to Docker Hub
             - run:
                 command: |
+                    source $BASH_ENV
```
Comment (dkontorovskyy) [inline .circleci/config.yml:467]: Why we need to source it everytime? Isn't it's sources by default on each command?

Comment (ivanstanev) [reply .circleci/config.yml:467]: No idea but it didn't work without it

File: .circleci/config/jobs/operator_upgrade_tests.yml [hunk 2]
```diff
@@ -14,30 +14,31 @@ steps:
   - run:
       name: Install required packages
       command: |
-        sudo apt install -y uuid-runtime
+        sudo apt update
+        sudo apt install -y uuid-runtime make
         python -m pip install requests pyyaml
-        python -m pip install operator-courier==2.1.7
+        python scripts/operator/download_operator_sdk.py
+        OPM_VERSION=v1.13.0 python scripts/operator/download_operator_package_manager.py
```
Comment (dkontorovskyy) [inline .circleci/config/jobs/operator_upgrade_tests.yml:21]: Why not make it as an env variable? I see here it's defind as `v1.13.0` and it scripts in try/except block it's `v.1.16.1`. Is this how it should be?

Comment (ivanstanev) [reply .circleci/config/jobs/operator_upgrade_tests.yml:21]: The machine executor uses an old debian version, the latest OPM 1.16.1 requires a later version of GLIBC that the current machine cannot have :/

Comment (ivanstanev) [reply .circleci/config/jobs/operator_upgrade_tests.yml:23]: Added a comment there

**Review by dkontorovskyy — APPROVED** [review]
LGTM

**Pull request merged**